### PR TITLE
feat: example_yamil

### PR DESCRIPTION
**Pull request by SebasthianSalpozz**
## Description

It's just an example of how our .yamil language can be implemented

## Type of Change

Put an `x` in all the boxes that apply:

- [x] ✨ New Feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug Fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking Change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code Refactor
- [ ] ✅ Build Configuration Change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

## Pull Request Checklist

Put an `x` in all the boxes that apply:

- [ ] My commit messages are detailed.
- [x] My code follows the code style of this project.
- [x] No existing features have been broken without good reason.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests pass.

### CODE REVIEW

**Review by fabianromeroclaros — APPROVED** [review]

File: resources/Example.yamil [hunk 1]
```diff
@@ -0,0 +1,48 @@
+def calculate(a: int, b: int) -> int = {
```
Comment (teran-joseluis) [inline resources/Example.yamil:1]: Update the path to app directory, so move it inside without resources directory. Due to in Haskell folder structure app directory refers to the external files can be passed to the program, I mean connect with impure world.

File: resources/Example.yamil [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        b
+    else
+        c
+}
+
+max_of_three(3, 7, 5) 
+max_of_three(10, 3, 5) 
+max_of_three(1, 2, 3)  
+
+def filter_greater_than(threshold: int, value: int) -> bool = {
```
Comment (teran-joseluis) [inline resources/Example.yamil:41]: Can you update the name threshold to a one more descriptive?

File: resources/Example.yamil [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    let is_greater: bool = lambda(x: int) -> bool = { x > threshold }
```
Comment (teran-joseluis) [inline resources/Example.yamil:42]: Can you update the lambda predicate to this one we have in our BNF?

`<lambda_expr> ::= "lambda" "(" <params> ")" "->" <expr>`

File: resources/Example.yamil [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    is_greater(value)
+}
+
+filter_greater_than(10, 15)  
+filter_greater_than(10, 5)  
+
```
Comment (teran-joseluis) [inline resources/Example.yamil:48]: Remove this empty line please.

**Review by teran-joseluis — CHANGES_REQUESTED** [review]

File: resources/Example.yamil (hunk 1 above)
Comment (SebasthianSalpozz) [reply resources/Example.yamil:1]: I don't understand your point of view where do you recommend me to move this Example ?

**Review by teran-joseluis — APPROVED** [review]
Nice

**Pull request merged**